app/controllers/Api.scala

```diff
@@ -0,0 +1,45 @@
+package controllers
```

Comment:
why change from `Application` to `Api`?

Reply:
It was clashing with `Application` in the Play API. I can try to put it back